src/gfnff/calculator.f90

```diff
@@ -36,6 +37,13 @@ module xtb_gfnff_calculator
    use xtb_gfnff_generator, only : TGFFGenerator
    use xtb_gfnff_eg
    implicit none
+   interface
+      subroutine generate_wsc(mol,wsc)
+        import :: TMolecule, tb_wsc
+        type(TMolecule), intent(inout) :: mol
+        type(tb_wsc),    intent(inout) :: wsc
+      end subroutine generate_wsc
+   end interface
```

Comment:
I know we are not consistent with the indentation everywhere, but the preference is 3 spaces.